scripts/universal_entry.cpp

```
@@ -0,0 +1,113 @@
+    extern "C" void (*__stop_##x##_init[])(void); \
+    int entry_##x(int argc, char* argv[]) { \
+        puts("Selected: " #x); \
+        unsigned count = __stop_##x##_init - __start_##x##_init; \
+        for (unsigned i = 0; i < count; i++) \
+        { \
+            __start_##x##_init[i](); \
+        } \
+        return Stockfish_##x::main(argc, argv); \
+    }
+
+DEFINE_BUILD(x86_64_bmi2)
```

Cppcheck (2.20.0)

[error] Subtracting pointers that point to different objects

(subtractPointers)